fe/fe-core/src/test/java/com/starrocks/planner/TestRuntimeFilterDescription.java

```diff
@@ -40,4 +40,25 @@ public void testPushAcrossExchangeNode() throws IOException {
             Assert.assertEquals(rf.toThrift().getBuild_join_mode(), rfJoinMode);
         }
     }
+
+    @Test
+    public void testIsLocalApplicable() throws IOException {
```

Comment:
no meaning UT

Reply:
A daily case will be constructed to cover this situation, this UT is just used to protect the isLocalApplicable from been modified mistakenly.